jetson/percep/artag_detector.cpp

```diff
@@ -148,8 +148,7 @@ void TagDetector::updateDetectedTagInfo(rover_msgs::Target *arTags, pair<Tag, Ta
     tags.buffer.push_back(0);
     tags.buffer.push_back(0);
 
-  for (uint i=0; i<2; i++)
```

Comment:
You shouldn't have made any changes to artag_detector.cpp or artag_detector.hpp. I'd guess that something went wrong in the merge to cause this to happen. Could you copy artag_detector.cpp and artag_detector.hpp from main and paste that code into these files and re-push? That should ensure not adjustments were made to these files.

Reply:
I got a warning when I was building and this change fixed the warning. Here is a picture what the warning was.
<img width="1770" alt="Screen Shot 2021-01-07 at 1 08 37 PM" src="https://user-images.githubusercontent.com/71610772/103928293-a160a680-50e9-11eb-9064-06975cb8e3a4.png">

Reply:
I see. You did it to make sure there's no compile errors. That makes sense